### ISSUE TITLE
make build fails on new mac

### ISSUE THREAD

**Issue by pdmthorsrud** [fredrikhgrelland/vagrant-hashistack#382]
```
==> vagrant.hashistack: Creating a Vagrantfile in the build directory...
==> vagrant.hashistack: Adding box using vagrant box add ...
    vagrant.hashistack: (this can take some time if we need to download the box)
==> vagrant.hashistack: Calling Vagrant Up (this can take some time)...
==> vagrant.hashistack: destroying Vagrant box...
==> vagrant.hashistack: Deleting output directory...
Build 'vagrant.hashistack' errored after 8 seconds 377 milliseconds: ssh-config command returned error: The provider for this Vagrant-managed machine is reporting that it
is not yet ready for SSH. Depending on your provider this can carry
different meanings. Make sure your machine is created and running and
try again. Additionally, check the output of `vagrant status` to verify
that the machine is in the state that you expect. If you continue to
get this error message, please view the documentation for the provider
you're using.

```
![image](https://user-images.githubusercontent.com/51820995/95325544-35567400-0856-11eb-8d15-36ae573c97b4.png)

investigating right now

**Comment by Neha-Sinha2305** [comment]
I have see this before. Have you exported the certs(behind the proxy stuff)?

**Comment by pdmthorsrud** [comment]
I assume this is because I am behind a transparent proxy. Not quite sure where my certificates are physically stored because I am on mac, which uses its own Keychain, trying to figure it out atm

**Comment by pdmthorsrud** [comment]
So on my second run everything worked the way it should. Steps to fix cert problems when behind transparent proxy on MacOS:

1. Export all certs in keychain to file with `security find-certificate -a -p > certs.pem`
2. Set `CURL_CA_BUNDLE` and `SSL_CERT_FILE` equal to the path of `certs.pem` created in the previous step

**Comment by pdmthorsrud** [comment]
This has been fixed, and was added to README by #389. Seems I forgot to close this issue, so I'll do that now 😀
